# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/ndctl/hotfix-libtracefs.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- ndctl-80/cxl/json.c.vanilla	2024-12-09 17:38:15.836666679 +0100
+++ ndctl-80/cxl/json.c	2024-12-09 17:38:24.110000009 +0100
@@ -9,7 +9,7 @@
 #include <json-c/json.h>
 #include <json-c/printbuf.h>
 #include <ccan/short_types/short_types.h>
-#include <tracefs/tracefs.h>
+#include <libtracefs/tracefs.h>
 
 #include "filter.h"
 #include "json.h"
